consensus/src/counters.rs

```diff
@@ -843,6 +843,16 @@ pub static UNEXPECTED_PROPOSAL_EXT_COUNT: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Histogram for the number of txns to be executed in a block.
+pub static NUM_TXNS_TO_EXECUTE_PER_BLOCK: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        "quorum_store_with_limit_num_txns_to_execute_per_block",
```

Comment:
nit: just `num_txns_to_execute_per_block` ?

Reply:
Good catch, actually this has to be changed to "max_txns_from_block_to_execute" to be inline with the naming used in the PR